mariadb.html.md.erb

```diff
@@ -47,4 +49,23 @@ Cloud Foundry: [Spring Music Example](https://github.com/cloudfoundry-samples/sp
 
 ## <a id='limitations'></a> Known Limitations
 
-The base of our HA MariaDB offering is built by a galera cluster, which brings some [known limitations](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it.
+Our HA MariaDB offering is based on Galera cluster, which brings some [known limitations](https://mariadb.com/kb/en/mariadb/mariadb-galera-cluster-known-limitations/) with it.
```

Comment:
in my ears "based on Galera cluster" sounds correct. Keep in mind that Galera is a plugin (wsrep) for MariaDB.